### ISSUE TITLE
Review usability for student groups

### ISSUE THREAD

**Issue by ddprrt** [splace-mag/framework#1]
Pinging @steima 
This should be the first draft. Would love to iterate over the framework. Folder- and filestructure for students should stay the same

**Comment by steima** [comment]
Definitely! I would like to create a demo widget which the students can use as their basis. Perhaps we should meet F2F some time next week?
